tests/unit/components/playback/controls/SettingsFormVuex.spec.js

```diff
@@ -0,0 +1,203 @@
+import { mount } from "@vue/test-utils";
+import ComponentToTest from "@/components/playback/controls/player/SettingsForm.vue";
+import { SettingsForm as DistComponentToTest } from "@/dist/mantarray.common";
+
+import Vuex from "vuex";
+import { createLocalVue } from "@vue/test-utils";
+import BootstrapVue from "bootstrap-vue";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+let wrapper = null;
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
+localVue.use(BootstrapVue);
+
+let NuxtStore;
+let store;
+
+describe("SettingsForm.vue", () => {
+  beforeAll(async () => {
+    // note the store will mutate across tests, so make sure to re-create it in beforeEach
+    const storePath = `${process.env.buildDir}/store.js`;
+    NuxtStore = await import(storePath);
+  });
+
+  beforeEach(async () => {
+    store = await NuxtStore.createStore();
+  });
+
+  afterEach(() => wrapper.destroy());
+  const array_of_userid_1 = [
+    {
+      user_id: 0,
+      uuid: "2VSckkBYr2An3dqHEyfRRE",
+      nickname: "User account -1",
+    },
+    {
+      user_id: 1,
+      uuid: "5FY8KwTsQaUJ2KzHJGetfE",
+      nickname: "User account -2",
+    },
+  ];
+  const array_of_userid_2 = [
+    {
+      user_id: 0,
+      uuid: "2VSckkkkk2An3dqHEyfRRE",
+      nickname: "Lab User -1",
+    },
+    {
+      user_id: 1,
+      uuid: "5FY8ghtsQaUJ2KzHJGetfE",
+      nickname: "Intern -1",
+    },
+  ];
+  const array_of_customerids = [
+    {
+      cust_id: 0,
+      uuid: "4vqyd62oARXqj9nRUNhtLQ",
+      api_key: "941532a0-6be1-443a-a9d5-d57bdf180a52",
+      nickname: "Customer account -1",
+      user_ids: array_of_userid_1,
+    },
+    {
+      cust_id: 1,
+      uuid: "6cBaidlJ84Ggc5JA7IYCgv",
+      api_key: "941532a0-6be1-443a-cdee-d57bdf180a52",
+      nickname: "Customer account -2",
+      user_ids: array_of_userid_2,
+    },
+  ];
+  const array_of_special_userids = [
```

Comment:
"special" is vague.  What exactly about this array of user IDs makes it different from the others?

Reply:
Corrected to `array_of_customer_ids_missing_user_ids` and  `array_of_customerid_null_missing_user_ids`  the word special is removed.